coral-service/src/main/java/com/linkedin/coral/coralservice/controller/TranslationController.java

```diff
@@ -117,6 +122,7 @@ public ResponseEntity getIncrementalInfo(@RequestBody IncrementalRequestBody inc
     incrementalResponseBody.setIncrementalQuery(null);
     try {
       if (language.equalsIgnoreCase("spark")) {
+        // TODO: replace getSparkIncrementalQueryFromUserSql with getIncrementalQuery
         String incrementalQuery = getSparkIncrementalQueryFromUserSql(query);
```

Comment:
How does this work when the source/target specified by the user is not Spark?

Reply:
`getIncrementalQuery(String query, String sourceLanguage, String targetLanguage)`  takes in a sourceLanguage param.

Reply:
Still not quite clear. Why are not we using `getIncrementalQuery` here and leaving it as a TODO?

Reply:
Can we hardcode "spark" and get rid of `getSparkIncrementalQueryFromUserSql`? The TODO will look different as well.